webapp/app/js/directives/clonejobs.js

```diff
@@ -93,6 +93,112 @@ treeherder.directive('thCloneJobs', [
     });
 
     $rootScope.$on(
+        thEvents.selectNextJob, function(ev){
+
+        var jobMap = ThResultSetModel.getJobMap($rootScope.repoName);
+        var lastJobSelected = ThResultSetModel.getSelectedJob($rootScope.repoName);
+        var el, key, job, step;
+
+        step = 'nextElementSibling';
+
+        // No job is selected currently , no job is selected till now ( 2 different cases )
+        if (lastJobSelected.el == undefined || lastJobSelected.el[0] == undefined ) {
+
+            // select first Job
```

Comment:
Leading capital in comments is a good idea, and in any other comments that have been added in the changes. I'm not sure why Job is capitalized here since it isn't anywhere else. eg. per my previous feedback:

`// My comment`